contracts/linear/src/events.rs

```diff
@@ -122,6 +122,11 @@ pub enum Event<'a> {
         old_weight: u16,
         new_weight: u16,
     },
+    ValidatorUpdatedWeights {
```

Comment:
we don't need to keep both `ValidatorUpdatedWeight` and `ValidatorUpdatedWeights` I think (since it's not used anywhere afaik), maybe we can replace the old one with the new one, and bump event standard version as well.